### DISCUSSION TITLE
Can i join team?

### DISCUSSION THREAD

**Discussion by AbrarNoobAlt** [ROM-builders/temporary#17979] [General]
Git Username : AbrarNoobAlt

Dt: https://github.com/AbrarNoob/device_xiaomi_alioth/tree/banana

https://github.com/AbrarNoob/device_xiaomi_sm8250-common/tree/banana

Successful Build:
https://cirrus-ci.com/build/4528045877362688

**Comment by Apon77** [comment]
done

**Reply by AbrarNoob** [reply]
Thanks 😊
